store/helpers/pages/pages.js

```diff
@@ -0,0 +1,97 @@
+import getOrder from '../order'
+import Page from './page'
+import invalidatePreviousRequests from '../invalidatePreviousRequests'
+
+import { LinkedList } from 'utils/linkedList'
+import apiRequest from 'api'
+import { CancelablePromise } from 'utils/promise'
+
+const PAGE_SIZE = 100
+
+export default class Pages {
+  #pages = new LinkedList()
+
+  #searchTerm = ''
+
+  #columnOrder = ''
+
+  #pageNumber = 0
+
+  isLastPageLoaded = false
+
+  isFirstPageLoaded = false
```

Comment:
If you go to private properties, this technically should be a getter of another private property.

Sometimes JavaScript gives you and everybody a freedom that if you want to take off you should spend a lot of effort. I prefer to do not do it at all.

Reply:
> If you go to private properties, this technically should be a getter of another private property.
I am not an expert in object-oriented design but I don't see any reason for doing so.